src/app/Http/Controllers/ArticleController.php

```diff
@@ -27,9 +32,16 @@ public function create(): Factory|View|Application|RedirectResponse
     public function store(ArticleRequest $request): Factory|View|Application|RedirectResponse
     {
         $article = Article::create($request->validated());
```

Comment:
`$request->validated()` has many fields which are not on the article table. `Article::create()` is the last step.